Assets/HoloToolkit-Examples/SpatialMappingComponent/DropCube.cs

```diff
@@ -11,22 +11,22 @@ namespace HoloToolkit.Examples.SpatialMappingComponent
     /// </summary>
     public class DropCube : MonoBehaviour
     {
-        GestureRecognizer recognizer;
+        private GestureRecognizer recognizer;
 
         private void Start()
         {
             recognizer = new GestureRecognizer();
             recognizer.SetRecognizableGestures(GestureSettings.Tap);
-            recognizer.OnTappedEvent += Recognizer_OnTappedEvent;
```

Comment:
Seriously Unity? Wow! I think we have changed these with pretty much every Unity release.